Refactor docs to remove reference to file UI encrypting (#398)
codemagic-ci-cd/codemagic-docs: content/building/encrypting.md

Encrypted(Z0FBQUFBQmRyY1FLWXIwVEhqdWphdjRhQ0xubkdoOGJ2bThkNmh4YmdXbFB3S2wyNTN2OERoV3c0YWU0OVBERG42d3Rfc2N0blNDX3FfblZxbUc4d2pWUHJBSVppbXNXNC04U1VqcGlnajZ2VnJVMVFWc3lZZ289)
```

> If you wish to encrypt a **file** to add to your workflow, you will first have to base64 encode it, and then encrypt the received string. To use the file you will have to decode it during the build.

Reviewer: ```suggestion
If you wish to encrypt a **file** to add to your workflow, you will first have to base64-encode it and then encrypt the received string. To use the file you will have to decode it during the build.
```

Reviewer: Is it important to clarify at what stage the decoding should happen in the build? That is where in the yaml they should have the line to decode the file?

Reviewer: Ok, I saw you mention later in the doc that it should be done in scripts.

> On MacOS, in order to base64 encode a file and copy the contents to your clipboard, you can run the following command in the terminal:

Reviewer: ```suggestion
On macOS, in order to base64 encode a file and copy the contents to your clipboard, you can run the following command in the terminal:
```

> With MacOS, in order to base64 encode a file and copy the contents to your clipboard, you can run the following command in the terminal:

Reviewer: With MacOS > On macOS

  ```bash
  cat your_file_name.extension | base64 | pbcopy 
  ```

> After the command has been run, you can paste the string from your clipboard to our encryption tool and then save the encrypted string to your environment variables.

Reviewer: ... to an environment variable.
```suggestion
After the command has been run, you can paste the string from your clipboard to our encryption tool and then save the encrypted string to an environment variable.
```

> Once done, you can base64 decode it during build time in your scripts section using the following command:

Reviewer: ```suggestion
Finally, base64 decode it during build time in your scripts section using the following command:
```
